osquery/main/harnesses/gen_fuzz_config_dict.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# If this is not commented out, we get 'sed' commands in our output below
+#set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function usage() {
+    echo "${BASH_SOURCE[0]} destination-file"
+}
+
+function main() {
+    if [[ $# < 1 ]]; then
```

Comment:
What's the expected usage? This seem like it's mean to be called with an argument for where output goes? I generally expect scripts to take arguments for input and output to stdout

Reply:
These scripts will be called as part of the [oss-fuzz build.sh](https://github.com/google/oss-fuzz/blob/master/projects/osquery/build.sh) file to put the resulting files in the $OUT dir with the appropriate name. [jbig2dec demonstrates what the files are supposed to look like](https://github.com/google/oss-fuzz/blob/b70dfdbb22515af6e6cbef23a5a26d0f2c50febe/projects/jbig2dec/build.sh) (although I don't know why it calls `unzip -l`).

Reply:
As far as taking an argument for output; for the .dict file I can print to stdout and pipe to a file if you like; but for the corpus the output needs to be a zip file. (And my initial thought was to make the 'one argument which is the output location' a uniform calling convention.)